cfn-resources/ldap-configuration/cmd/resource/resource.go

```diff
@@ -36,12 +38,12 @@ const (
 	Port         = "Port"
 )
 
-var CreateRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.GroupID,
+var CreateRequiredFields = []string{constants.GroupID,
 	BindUsername, BindPassword, Hostname, Port}
-var ReadRequiredFields = []string{constants.GroupID, constants.PubKey, constants.PvtKey}
-var UpdateRequiredFields = []string{constants.GroupID, constants.PubKey, constants.PvtKey}
-var DeleteRequiredFields = []string{constants.GroupID, constants.PubKey, constants.PvtKey}
-var ListRequiredFields = []string{constants.PubKey, constants.PvtKey}
+var ReadRequiredFields = []string{constants.GroupID}
+var UpdateRequiredFields = []string{constants.GroupID}
+var DeleteRequiredFields = []string{constants.GroupID}
```

Comment:
From the other PRs, we've been standardising on ProjectID over GroupID. Good to be consistent, especially as these changes would be a breaking change in the future.